src/content/en/fundamentals/security/credential-management/retrieve-credentials.md

```diff
@@ -342,9 +353,11 @@ Learn more about this information in the
 [Credential Management specification](https://w3c.github.io/webappsec-credential-management/#credential).
 
 To store federated account details, instantiate a new 
-[`FederatedCredential`](https://developer.mozilla.org/en-US/docs/Web/API/FederatedCredential), object with the user's identifier and the provider's identifier.
+[`FederatedCredential`](https://developer.mozilla.org/en-US/docs/Web/API/FederatedCredential),
+object with the user's identifier and the provider's identifier.
 Then invoke
-[`navigator.credentials.store()`](https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/store) to store the identity information.
+[`navigator.credentials.store()`](https://developer.mozilla.org/en-US/docs/Web/API/CredentialsContainer/store)
+to store the identity information.
 
 For example:
 
```

Comment:
I would like to have `navigator.credentials.create()` example here as well. The existing one is a sync example but additional one would be an async example. (Note that this code example is slightly different from what we have in [the update article](https://developers.google.com/web/updates/2017/06/credential-management-updates), but this one is correct.)

```
var cred = await navigator.credentials.create({
  federated: {
    id:       id,
    provider: 'https://accounts.google.com',
    name:     name,
    iconURL:  iconUrl
  }
});